modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
 	compileOnly project(":core:petra:petra-function")
 	compileOnly project(":core:petra:petra-lang")
 	compileOnly project(":core:petra:petra-reflect")
+	compileOnly project(":core:petra:petra-sql-dsl-api")
```

Comment:
@MrJohn1911 Why this is needed?

Reply:
Because of this compile error:

[exec] > Task :apps:dynamic-data-mapping:dynamic-data-mapping-form-builder:compileJava FAILED
     [exec] /opt/dev/projects/github/liferay-portal/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMDataProviderInstancesServlet.java:100: error: cannot access com.liferay.petra.sql.dsl.query.sort.OrderByInfo
     [exec]       DataProviderInstanceNameComparator.get(true);
     [exec]                                         ^
     [exec]   class file for com.liferay.petra.sql.dsl.query.sort.OrderByInfo not found